src/components/contrib/controlbar.scss

```diff
@@ -0,0 +1,318 @@
+.hide {
```

Comment:
remove this file should not be referenced anymore

Reply:
done